### DISCUSSION TITLE
buf.yaml configure maximum line length

### DISCUSSION THREAD

**Discussion by myz-dev** [bufbuild/buf#3407] [Q&A] [answered]
Hi,

I am trying to disable the lint for exceeding 80 characters per line (or at least set a higher limit).
Either I am doing something completely wrong, or I am easily exceeding the limit when trying to adhere to the best practice of package naming and directory housekeeping. 
Neither at the [reference](https://buf.build/docs/configuration/v2/buf-yaml/) nor at the [rules section](https://buf.build/docs/lint/rules/) did I find the explanation of a rule. Hopefully someone can help me.

**Comment by timostamm** [comment]
Could you paste the full error message here?

**Reply by myz-dev** [reply]
Thank you for the swift reply. There is no error message, just the expected lint:

For example: `The line length is 89, but it must be shorter than 80`. I wonder if I can turn this lint off or configure it in `buf.yaml` so I can give it a higher limit of characters.

**Reply by timostamm** [reply, marked as answer]
This isn't a lint rule enforced by buf. You can check by running the `buf lint` command (see the [reference here](https://buf.build/docs/reference/cli/buf/lint/)). Maybe some other linter is running at the same time?

**Reply by myz-dev** [reply]
Okay thank you. Running it in the command line indeed does not print out anything. I have yellow lines in my editor, I will see if I mistakenly installed any other linter with `buf`.
Thank you already for your support!
